src/ceph_osd.cc

```diff
@@ -563,6 +573,9 @@ int main(int argc, const char **argv)
   r = ms_hb_front_server->bind(hb_front_addr);
   if (r < 0)
     exit(1);
+  r = ms_hb_front_client->client_bind(hb_back_addr);
```

Comment:
@yuriw sorry, I have a mistake. you may need to update this pr to branch and schedule qa